docs/en/rules/Azure.ACR.ExportPolicy.md

```diff
@@ -0,0 +1,112 @@
+---
+reviewed: 2025-07-09
+severity: Important
+pillar: Security
+category: SE:08 Hardening resources
+resource: Container Registry
+resourceType: Microsoft.ContainerRegistry/registries
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.ACR.ExportPolicy/
+---
+
+# Container Registry export policy should be disabled
+
+## SYNOPSIS
+
+Disable export of artifacts from Azure container registry to ensure data is accessed solely via the data plane.
```

Comment:
In one sentence, update this synopsis to reflect what the problem is instead of what the resolution is.